src/main/java/org/jvnet/hudson/test/RealJenkinsRule.java

```diff
@@ -359,12 +360,12 @@ public void startJenkins() throws Throwable {
             throw new IllegalStateException("Jenkins is (supposedly) already running");
         }
         String cp = System.getProperty("java.class.path");
+        FileUtils.writeLines(new File(home, "RealJenkinsRule-cp.txt"), Arrays.asList(cp.split(File.pathSeparator)));
         List<String> argv = new ArrayList<>(Arrays.asList(
                 new File(System.getProperty("java.home"), "bin/java").getAbsolutePath(),
                 "-ea",
                 "-Dhudson.Main.development=true",
                 "-DRealJenkinsRule.location=" + RealJenkinsRule.class.getProtectionDomain().getCodeSource().getLocation(),
-                "-DRealJenkinsRule.cp=" + cp,
                 "-DRealJenkinsRule.url=" + getUrl(),
                 "-DRealJenkinsRule.description=" + description,
                 "-DRealJenkinsRule.token=" + token));
```

Comment:
Alternately, could write all of these to `$JENKINS_HOME/RealJenkinsRule.properties` and have `Init2` read the whole thing and bind to system properties.